proto/viam/app/v1/app.proto

```diff
@@ -854,14 +861,17 @@ message ListFragmentsRequest {
 
 message ListFragmentsResponse {
   repeated Fragment fragments = 1;
+  repeated FragmentUsage fragment_usages = 2;
 }
 
 message GetFragmentRequest {
   string id = 1;
+  string organization_id = 2;
```

Comment:
This is a bit weird since in most APIs the `organization_id` is the owner of the entity. This is really `current_organization_id` so lets name as such in my opinion. And I think it makes sense to make `machines_in_org` be `machines_in_current_org`

Reply:
i agree, we should probably move to `current_organization_id` in other endpoints too to be extra clear about this. updated both fields to have "current org" in it